pkg/metadata/inventories/README.md

```diff
@@ -74,6 +86,31 @@ The payload is a JSON dict with the following fields
     `compliance_config.enabled` config option).
   - `feature_apm_enabled` - **bool**: True if the APM Agent is enabled (see: `apm_config.enabled` config option).
   - `feature_otlp_enabled` - **bool**: True if the OTLP pipeline is enabled.
+- `host_metadata` - **dict of string to JSON type**:
+  - `cpu_cores` - **int**: the number of core for the host.
+  - `cpu_logical_processors` - **int**:  the number of logical core for the host.
+  - `cpu_vendor` - **string**: the CPU vendor.
+  - `cpu_model` - **string**:  the CPU model.
+  - `cpu_model_id` - **string**: the CPU model ID.
+  - `cpu_family` - **string**: the CPU family.
+  - `cpu_stepping` - **string**: the CPU stepping.
+  - `cpu_frequency` - **number/float**: the CPU frequency.
+  - `cpu_cache_size` - **int**: the CPU cache size in bytes (only fill on Linux only, 0 for Windows and OSX).
+  - `cpu_architecture` - **string**: the hardware name, Linux only (ex "x86_64", "unknown", ...).
+  - `kernel_name` - **string**: the kernel name (ex: "windows", "Linux", ...).
+  - `kernel_release` - **string**:  the kernel release (ex: "10.0.20348", "4.15.0-1080-gcp", ...).
+  - `kernel_version` - **string**:  the kernel version (Unix only, empty string on Windows).
+  - `os` - **string**: the OS name description (ex: "GNU/Linux", "Windows Server 2022 Datacenter", ...).
+  - `os_version` - **string**: the OS version (ex: "debian bookworm/sid", ...).
+  - `python_version` - **string**: The Python version from the agent environment. `python -V` is used in `Gohai` for
+    this. Unless the Agent environment has been modified this is the Python version from the OS not from the Agent.
```

Comment:
This seems contradictory ("agent environment" vs. "from the OS").  I suspect this value will be confusing as often as it's useful.  Would it make sense to just omit it?